common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3190,6 +3190,11 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "has had a transaction done on it since the last major compaction. So decreasing this\n" +
         "value will increase the load on the NameNode."),
 
+    HIVE_COMPACTOR_INITIATOR_DURATION_UPDATE_INTERVAL("hive.compactor.initiator.duration.update.interval", "1s",
+        new TimeValidator(TimeUnit.SECONDS),
+        "Time in seconds that drives the update interval of compaction_initiator_duration metric.\n" +
+            "Smaller value results in a fine grained metric update.\n"),
```

Comment:
Do you mind adding a sentence here, what will happen if this value is negative? Please mention as well, that `hive.compactor.initiator.on` must be set to `true` to make this work.